addons/mail/static/tests/discuss/typing/typing.test.js

```diff
@@ -347,3 +347,38 @@ test("chat: correspondent is typing in chat window", async () => {
     );
     await contains("[title='Demo is typing...']", { count: 0 });
 });
+
+test("show typing status of members in member list", async () => {
+    const pyEnv = await startServer();
+    const userId = pyEnv["res.users"].create({ name: "Other 10" });
+    const partnerId = pyEnv["res.partner"].create({ name: "Other 10", user_ids: [userId] });
+    const channelId = pyEnv["discuss.channel"].create({
+        name: "channel",
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId }),
+            Command.create({ partner_id: partnerId }),
+        ],
+    });
+    await start();
+    await openDiscuss(channelId);
+    await click(".o-mail-Discuss-header button[name='member-list']");
+    await contains(".o-discuss-ChannelMember", { count: 2 });
+    withUser(userId, () =>
+        rpc("/discuss/channel/notify_typing", {
+            channel_id: channelId,
+            is_typing: true,
+        })
+    );
+    await contains(
+        ".o-discuss-ChannelMember .o-discuss-Typing-icon[title='Other 10 is typing...']"
+    );
+    withUser(serverState.userId, () =>
+        rpc("/discuss/channel/notify_typing", {
+            channel_id: channelId,
+            is_typing: true,
+        })
+    );
+    await contains(
+        `.o-discuss-ChannelMember .o-discuss-Typing-icon[title='${serverState.partnerName} is typing...']`
```

Comment:
`.o-discuss-ChannelMemberList [title='']` is good enough. Exact placement is not really necessary for coverage of this test. This is more test maintenance and makes it harder to read when just caring this info is there is good enough.